src/services/api.ts

```diff
@@ -230,4 +300,14 @@ export class Api {
   ): Promise<BigNumber> {
     return getExecutedSellAmount({ ...this.apiCallParams(), ...query });
   }
+  async getFeeAndQuoteSell(
```

Comment:
At first I did that, but then I thought that switching from a buy to a sell order would also require the surrounding logic to change and not just a change in how the api is called because of how the fee needs to be handled.
Overall I think the current api syntax makes sense and having destructuring/Typescript explicitly tell you if the amounts include fees or not could be helpful.